johnsonandjohnson/Bodiless-JS
# docs(bodiless-i18n): API doc and implementation Docs.

## Changes

- Update files to build API doc and have available on bodiless test site & vital demo site.
- Update API doc with categories
- Add Readme
- zero code changes, except for non-risk export of some api/tokens.

## Test Instructions
- Verify http://localhost:8000/___docs/#/Development/API/@bodiless/i18n/README loads and looks nice.
- Verify http://localhost:8000/___docs/#/Components/Internationalization/ loads and looks nice
